### PR TITLE
DOC-7211 export metrics|logs no creation date restriction

### PR DESCRIPTION
**Pull request by andf-crl**
Addresses: DOC-7211

- Removed mentions of creation date restriction from both `export-logs.md` and `export-metrics.md`.

Timing

- I will wait to merge until C C - 9 7 7 5 is merged.

Staging

[export-logs.md](https://deploy-preview-16516--cockroachdb-docs.netlify.app/docs/cockroachcloud/export-logs.html) | [export-metrics.md](https://deploy-preview-16516--cockroachdb-docs.netlify.app/docs/cockroachcloud/export-metrics.html)

### CODE REVIEW

**Review by thtruo — APPROVED** [review]
Nice @andf-crl this LGTM!

**Review by MichaelTrestman — APPROVED** [review]
lgtm

File: cockroachcloud/export-logs.md [hunk 1]
```diff
@@ -8,10 +8,6 @@ cloud: true
 
 {{ site.data.products.dedicated }} users can use the [Cloud API](cloud-api.html) to configure log export to [AWS CloudWatch](https://aws.amazon.com/cloudwatch/) or [GCP Cloud Logging](https://cloud.google.com/logging). Once the export is configured, logs will flow from all nodes in all regions of your {{ site.data.products.dedicated }} cluster to your chosen cloud log sink. You can configure log export to redact sensitive log entries, limit log output by severity, send log entries to specific log group targets by log channel, among others.
 
-{{site.data.alerts.callout_danger}}
-The {{ site.data.products.dedicated }} log export feature is only available on clusters created after August 11, 2022 (AWS) or September 9, 2022 (GCP).
```
Comment (abarganier) [inline cockroachcloud/export-logs.md:12]: Note: the limitation was removed for **GCP only**. AWS still has this limitation. We're working on fixing this right now, but until it's merged, deployed, and run, the AWS limitation still exists.

Can we rework this to only remove the GCP bit, leaving the AWS piece?

**Review by abarganier — CHANGES_REQUESTED** [review]

File: cockroachcloud/export-metrics.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ cloud: true
 
 {{ site.data.products.dedicated }} users can use the [Cloud API](cloud-api.html) to configure metrics export to [AWS CloudWatch](https://aws.amazon.com/cloudwatch/) or [Datadog](https://www.datadoghq.com/). Once the export is configured, metrics will flow from all nodes in all regions of your {{ site.data.products.dedicated }} cluster to your chosen cloud metrics sink.
 
-Exporting metrics to AWS CloudWatch is only available on {{ site.data.products.dedicated }} clusters which are hosted on AWS, and were created after August 11, 2022. Metrics export to Datadog is supported on all {{ site.data.products.dedicated }} clusters regardless of creation date.
```
Comment (abarganier) [inline cockroachcloud/export-metrics.md:11]: Note: the AWS limitation also still exists for metric export.

We're hoping to resolve this soon, but I think we should scope this PR to only removing the text about the GCP limitation, and then follow up with another PR once the AWS limitation is removed.

**Comment by thtruo** [comment]
Update: this is now complete https://cockroachlabs.atlassian.net/browse/CC-9775 so we can fully remove the mentions of creation date restriction from both `export-logs.md` and `export-metrics.md`.

**Comment by thtruo** [comment]
One more request: in https://www.cockroachlabs.com/docs/cockroachcloud/export-logs.html we mention that Log Export is in Limited Preview. Now that we removed this creation date limitation for older clusters, we can treat Log Export as a Preview feature. Can we use the same language as in our Metric Export doc https://www.cockroachlabs.com/docs/cockroachcloud/export-metrics.html?

> This feature is in [preview](https://www.cockroachlabs.com/docs/v23.1/cockroachdb-feature-availability). This feature is subject to change. To share feedback and/or issues, contact [Support](https://support.cockroachlabs.com/hc/en-us).

**Comment by florence-crl** [comment]
I could not merge this PR. After consulting with @lnhsingh, I created and merged #17241.

**Pull request closed without merging**